bot.js

```diff
@@ -1,197 +1,75 @@
 require("dotenv").config();
 
 let TelegramBot = require("node-telegram-bot-api");
+let game = require("./game.js");
 
 if (!process.env.TELEGRAM_TOKEN) {
   throw new Error("TELEGRAM_TOKEN env variable is missing");
 }
 
 const bot = new TelegramBot(process.env.TELEGRAM_TOKEN, { polling: true });
-let gameStart = false;
-let spentCities = new Set();
-let lastLetter = "";
+const usedKeyWords = ["/start", "начать", "сдаюсь"];
```

Comment:
You don't need this anymore. You have all the commands in the `commands` object. Simply check that your message is not matching with any of regular expressions